NAMESPACE

```diff
@@ -706,7 +706,6 @@ export(path.length.hist)
 export(permute)
 export(permute.vertices)
 export(piecewise.layout)
-export(plot.igraph)
```

Comment:
oops, not sure why running `devtools::document()` would remove those? or was `NAMESPACE` out of date?

Reply:
Interesting (for some definition of).

I wonder if this still happens if we keep the source files that define `plot.igraph()` and `print.igraph()` unchanged.